docs/doc/30-reference/30-sql/20-query-syntax/dml-join.md

```diff
@@ -2,17 +2,52 @@
 title: JOIN
 ---
 
-A *join* allows you to combine columns from two or more tables into a single result set. Databend supports the following *join* types:
+## Supported Join Types
 
-* Inner Join
-* Natural Join
-* Cross Join
-* Left Join
-* Right Join
+The *join* combines columns from two or more tables into a single result set. Databend supports the following *join* types:
+
+* [Inner Join](#inner-join)
+* [Natural Join](#natural-join)
+* [Cross Join](#cross-join)
+* [Left Join](#left-join)
+* [Right Join](#right-join)
+* [Full Outer Join](#full-outer-join)
+* [Left / Right Semi Join](#left--right-semi-join)
+* [Left / Right Anti Join](#left--right-anti-join)
+
+## Example Tables
+
+Unless explicitly specified, the join examples on this page are created based on the following tables:
+
+Table "vip_info": This table stores the VIP client information.
```

Comment:
👍, nice to see the example put in the front.

Reply:
Thank you